src/RoadRegistry.Hosts/RoadRegistryHostBuilder.cs

```diff
@@ -159,6 +162,18 @@ private RoadRegistryHostBuilder<T> ConfigureDefaultLogging()
                 .Enrich.WithThreadId()
                 .Enrich.WithEnvironmentUserName();
 
+            var slackSinkConfiguation = hostContext.Configuration.GetSection(nameof(SlackSinkOptions));
+
+            if (!slackSinkConfiguation.Exists()) return;
```

Comment:
Die `return` hier mag niet, de rest van de log config moet wel nog gebeuren :)
Kan dit anders in een extension method (dan kan API die ook gebruiken) en dan is de return wel ok

Reply:
Added through extension method instead located inside Host project